libs/sdk-ui-web-components/README.md

```diff
@@ -0,0 +1,130 @@
+# GoodData Web Component
+
+This library provides a set of Custom Elements (aka Web Components) to easy the embedding of GoodData visualizations
```

Comment:
The formulation is a bit strange, how about
> This library provides a set of Custom Elements (aka Web Components) to make the embedding of GoodData visualizations into web applications easier.

Reply:
Yeah, it should have been "ease", not "easy" :) But I like your option better,